src/site/content/en/blog/maskable-icon/index.md

```diff
@@ -76,26 +97,51 @@ If you want to create a maskable icon based off your existing icon, you can use
   </figcaption>
 </figure>
 
-Once you've created a maskable icon image and tested it out in DevTools, you'll need to update your [Web App Manifest](https://developers.google.com/web/fundamentals/web-app-manifest) to point to the new assets. The Web App Manifest provides information about your web app in a JSON file, and includes an [`icons` array](https://developers.google.com/web/fundamentals/web-app-manifest#icons).
-
-With the inclusion of maskable icons, a new property value has been added for image resources listed in a Web App Manifest. The `purpose` field tells the browser how your icon should be used. By default, icons will have a purpose of `"any"`. These icons will be resized on top of a white background on Android.
-
-Maskable icons should use a different purpose: `"maskable"`. This indicates that an image is meant to be used with icon masks, giving you more control over the result. This way, your icons will not have a white background. You can also specify multiple space-separated purposes (for example, `"any maskable"`), if you want your maskable icon to be used without a mask on other devices.
+Once you've created a maskable icon image and tested it out in DevTools, you'll need to update your
+[Web App Manifest](https://developers.google.com/web/fundamentals/web-app-manifest) to point to the
+new assets. The Web App Manifest provides information about your web app in a JSON file, and
+includes an [`icons` array](https://developers.google.com/web/fundamentals/web-app-manifest#icons).
+
+With the inclusion of maskable icons, a new property value has been added for image resources listed
+in a Web App Manifest. The `purpose` field tells the browser how your icon should be used. By
+default, icons will have a purpose of `"any"`. These icons will be resized on top of a white
+background on Android.
+
+Maskable icons should use a different purpose: `"maskable"`. This indicates that an image is meant
+to be used with icon masks, giving you more control over the result. This way, your icons will not
+have a white background. You can also specify multiple space-separated purposes (for example,
+`"any maskable"`), if you want your maskable icon to be used without a mask on other devices.
+
+{% Aside %}
+  While you _can_ specify multiple space-separated purposes like `"any maskable"`, in
+  practice you _shouldn't_, since using `"maskable"` icons as `"any"` icons is suboptimal as the icon
```

Comment:
```suggestion
  practice you _shouldn't_. Using `"maskable"` icons as `"any"` icons is suboptimal as the icon
```

Reply:
Give the reader a chance to 'breath' before moving to the explanation. Also having the recommendation stand alone gives it more force. I didn't start the next sentence with "This is because" because we expect the next sentence to be the reason and it fulfills that expectation.